src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -254,8 +296,12 @@ export default function ActionsPanel(): JSX.Element {
       data-testid={TestId.PANEL}
       title={generateTitle()}
       onRemovePanelQueryParams={handleRemovePanelQueryParams}
+      ref={scrollArea}
     >
       <p data-testid="actions-panel-unit-list">
+        {inlineErrors.some((error) => error)
+          ? generateInlineErrors(inlineErrors, scrollArea.current)
+          : null}
```

Comment:
This needs to be outside of the wrapping `<p>` as this is currently giving errors about nested `<p>` tags.